components/automate-ui/src/app/pages/+compliance/+scanner/containers/nodes-list/nodes-list.component.ts

```diff
@@ -79,13 +81,19 @@ export class NodesListComponent implements OnInit, OnDestroy {
     this.router.navigate([], {queryParams});
   }
 
-  onNodeResultsShow(_event, id) {
+  onNodeResultsShow(id: string): void {
     this.showNodeResults = true;
-    this.store.dispatch(actions.getNode(id));
+
+    this.nodesList.items.map(node => {
+      if ( node.id === id ) {
+        this.failedNode = node;
+      }
+    });
   }
 
   onNodeResultsHide(_event) {
     this.showNodeResults = false;
+    this.failedNode = null;
```

Comment:
Not sure this is completely necessary because `failedNode` gets overwritten by new info when another node is selected, but felt it would probably be best to clear it out just in case.